django/db/models/sql/compiler.py

```diff
@@ -957,7 +1007,7 @@ def execute_sql(self, result_type):
             if is_empty:
                 rows = aux_rows
                 is_empty = False
-        return rows
+        return (rows, returning_values)
```

Comment:
@shaib I'm resuming work on this as part of https://github.com/django/django/pull/5904
Could you please give me an example scenario when we could get multiple rows so I can test and fix the case?